pkg/apis/openebs.io/lvm/v1alpha1/lvmvolume.go

```diff
@@ -65,6 +65,8 @@ type VolumeInfo struct {
 	// +kubebuilder:validation:MinLength=1
 	VolGroup string `json:"volGroup"`
 
+	SnapName string `json:"snapName,omitempty"`
```

Comment:
why do we need this?

Reply:
This was added for clone support, same as from ZFS LocalPV

Reply:
removed.